pkg/reporter/reporter.go

```diff
@@ -149,6 +152,38 @@ func (tr PrettyReporter) Publish(_ context.Context, r report.Report) error {
 	}
 
 	_, err := fmt.Fprint(tr.out, table+footer+"\n")
+	if err != nil {
+		return fmt.Errorf("failed to write report: %w", err)
+	}
+
+	f := fixer.NewFixer()
+	f.RegisterFixes(fixes.NewDefaultFixes()...)
```

Comment:
We'll have to consider what to do here if we make the new directory-package-mismatch rule `custom` and not "default"... but let's just make a note on that for now.